packages/anoncreds/src/models/AnonCredsProofRequest.ts

```diff
@@ -0,0 +1,83 @@
+import type { AnonCredsRequestedPredicateOptions } from './AnonCredsRequestedPredicate'
+
+import { IndyRevocationInterval } from '@aries-framework/core'
+import { Expose, Type } from 'class-transformer'
+import { IsIn, IsInstance, IsOptional, IsString, ValidateNested } from 'class-validator'
+
+import { IsMap } from '../utils'
+
+import { AnonCredsRequestedAttribute } from './AnonCredsRequestedAttribute'
+import { AnonCredsRequestedPredicate } from './AnonCredsRequestedPredicate'
+import { AnonCredsRevocationInterval } from './AnonCredsRevocationInterval'
+
+export interface AnonCredsProofRequestOptions {
+  name: string
+  version: string
+  nonce: string
+  nonRevoked?: AnonCredsRevocationInterval
+  ver?: '1.0' | '2.0'
```

Comment:
I know this comes from the spec but it seems to me that will be very confusing:

- ver is an optional string, specifying the presentation request version
- version is a string set by the verifier, the version of the presentation request

Reply:
Yup that's very weird. We don't expose them in the AFJ API as 2.0 is the default I think when you don't provide it